packages/account/src/Components/address-details/__tests__/address-details.spec.tsx

```diff
@@ -45,7 +46,7 @@ describe('<AddressDetails/>', () => {
         onSave: jest.fn(),
         onSubmit: jest.fn(),
         onSubmitEnabledChange: jest.fn(),
-        selected_step_ref: { current: { isSubmitting: false } },
+        selected_step_ref: { current: { isSubmitting: false } } as React.RefObject<FormikProps<FormikValues>>,
```

Comment:
can we use a real ref instead of this?
nitpick
```suggestion
        selected_step_ref: react.useRef({isSubmitting: false}),
```

Reply:
This is not working